domains/araujo.localplayer.dev.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Domínio para projeto pessoal em construção",
+  "domain": "localplayer.dev",
+  "subdomain": "araujo",
+  "owner": {
+    "repo": "https://github.com/josearaujojr/josearaujojr.github.io",
+    "email": "learn.labaws.002@gmail.com"
+  },
+  "record": {
+    "CNAME": "josearaujojr.github.io"
```

Comment:
Unfinished site. The first PR requirement is such: 
![image](https://github.com/user-attachments/assets/489026d3-c4dc-429a-a0f2-a9bc471cc53f)